easycv/models/backbones/vitdet.py

```diff
@@ -0,0 +1,849 @@
+# --------------------------------------------------------
+# BEIT: BERT Pre-Training of Image Transformers (https://arxiv.org/abs/2106.08254)
```

Comment:
change copyright, keep the original copyright and add a link for original code,
add corresponding license in NOTICE file

Reply:
already change copyright